static/css/home.scss

```diff
@@ -0,0 +1,507 @@
+  margin-top: 30px;
+  transition-duration: 0.25s;
+  transition-property: box-shadow, border;
+  user-select: none;
+  width: 260px;
+}
+
+.button.primary {
+  background: #30e60b;
+  color: #fff;
+  text-align: left;
+
+  &:hover {
+    background: #20c60b;
+  }
+
+  &:active {
+    background: #10b60b;
+    box-shadow: rgba(0, 0, 0, 0.09) 0 3px 4px;
+  }
+
+}
+
+.download-firefox .button-icon {
+  -ms-flex: 0 0 57px;
+  background-image:   url("../img/landing-small-firefox-logo@2x.png");
+  background-position: 50%;
+  background-repeat: no-repeat;
+  background-size: 57px 59px;
+  flex: 0 0 57px;
+  height: 59px;
+  margin-right: 16px;
+  position: relative;
+}
+
+.download-firefox .button-icon-badge {
+  background-color: #fff;
+  background-image:   url("../img/landing-download-icon@2x.png");
+  background-position: 50%;
+  background-repeat: no-repeat;
+  background-size: 14px 14px;
+  border-radius: 50%;
+  bottom: 4px;
+  box-shadow: 0 2px 0 rgba(0, 0, 0, 0.2);
+  height: 16px;
+  padding: 6px;
+  position: absolute;
+  right: 2px;
+  transition-duration: 0.15s;
+  width: 16px;
+}
+
+.download-firefox .button-title {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 1em;
+  margin-bottom: 0.25em;
+}
+
+.download-firefox .button-description {
+  font-size: 18px;
+  font-weight: 300;
+  line-height: 1em;
+}
+
+/* Instruction Sections */
+
+section {
+  align-items: center;
+  display: flex;
+  min-height: 600px;
+}
+
+section:nth-child(even) {
+  background-color: #f7fdfe;
+}
+
+section .container {
+  max-width: 1100px;
+  padding: 40px 63px;
+}
+
+.section-content h3 {
+  font-size: 2.5em;
+  font-weight: 400;
+  margin: 0;
+}
+
+.section-content .bold {
+  font-weight: 500;
+}
+
+.section-content {
+  color: #4a4a4a;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  line-height: 1.61;
+  width: 36%;
+}
+
+.section-image {
+  background-size: contain;
+  display: flex;
+  height: auto;
+  justify-content: flex-end;
+  width: 50%;
+}
+
+#section-1 .section-image {
+  background-image:   url("../img/landing-screenshots_image_01.svg");
+  background-position: 100% center;
+  background-repeat: no-repeat;
+  height: 476px;
+}
+
+#section-2 .section-image {
+  background-image:   url("../img/landing-screenshots_image_02.svg");
+  background-position: 0% center;
+  background-repeat: no-repeat;
+  height: 500px;
+}
+
+#section-3 .section-image {
+  background-image:   url("../img/landing-screenshots_image_03.svg");
+  background-position: 100% center;
+  background-repeat: no-repeat;
+  height: 519px;
+}
+
+#section-4 .section-image {
+  background-image:   url("../img/landing-screenshots_image_04.svg");
+  background-position: 100% center;
+  background-repeat: no-repeat;
+  height: 533px;
+}
+
+#section-1 .section-content,
+#section-2 .section-image,
+#section-4 .section-content {
+  order: 1;
+}
+
+#section-1 .section-image,
+#section-2 .section-content,
+#section-4 .section-image {
+  order: 2;
+}
```

Comment:
Not sure if I'm missing something, but wouldn't this just overrule what you did on lines 316-320 above?